packages/rum-core/src/domain/getSelectorFromElement.ts

```diff
@@ -38,16 +55,37 @@ const UNIQUE_AMONG_CHILDREN_SELECTOR_GETTERS: SelectorGetter[] = [
   getTagNameSelector,
 ]
 
-export function getSelectorFromElement(targetElement: Element, actionNameAttribute: string | undefined) {
+export function getSelectorFromElement(
+  targetElement: Element,
+  actionNameAttribute: string | undefined,
+  { stopRecurringWhenUnique = false, targetMeaningfulElement = false, onlyPrefixWithSemanticTag = false } = {}
+) {
   let targetElementSelector = ''
   let element: Element | null = targetElement
+  const prefix = onlyPrefixWithSemanticTag ? getSemanticTagName : getTagNameSelector
+
+  if (targetMeaningfulElement && targetElement.closest) {
+    const closestClickableElement = targetElement.closest(MEANINGFUL_ELEMENTS_SELECTORS.join(','))
+    if (closestClickableElement) {
+      element = closestClickableElement
+    }
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** could we stick to either ‏clickable or meaningful naming? or is it worth differentiating the two?

Reply:
Nop no reason to differentiating the two